### DISCUSSION TITLE
Firmware on CVE-2020-11239 doesn't exists

### DISCUSSION THREAD

**Discussion by nicochidt** [github/securitylab#315] [General] [answered]
The documentation on the CVE-2020-11239 says it was tested on a Samsung Galaxy A71 with firmware version A715FXXUATJ2 which doesn't seem to be a version ever released by Samsung (as can be checked on https://www.sammobile.com/firmwares/archive/). 

Was this a typo? Can you point to the firmware used?

**Comment by m-y-mo** [comment, marked as answer]
Thanks for spotting this. It is indeed a typo, should be A715FXXU3ATJ2, e.g. this one:
https://www.sammobile.com/samsung/galaxy-a71/firmware/SM-A715F/BTU/download/A715FXXU3ATJ2/486490/
I'll correct it in the repo. Thanks.

**Reply by nicochidt** [reply]
Thanks a lot for the prompt response!
